### ISSUE TITLE
Bind doc/quit

### ISSUE THREAD

**Issue by Nephos** [AuthCr/qt-auth-api#6]
Bind the buttons to some signals/urls.

Also bind them to keyboard shortcuts

**Comment by Lucie-Dispot** [comment]
Fixed in 95c22196be1d870c296bd30cdd81297b5552cd7b

**Comment by Nephos** [comment]
Ctrl+Q don't work

**Comment by Nephos** [comment]
```diff
- <connections/>
+ <connections>
+  <connection>
+   <sender>actionQuit</sender>
+   <signal>triggered()</signal>
+   <receiver>MainWindow</receiver>
+   <slot>close()</slot>
+   <hints>
+    <hint type="sourcelabel">
+     <x>-1</x>
+     <y>-1</y>
+    </hint>
+    <hint type="destinationlabel">
+     <x>424</x>
+     <y>232</y>
+    </hint>
+   </hints>
+  </connection>
+ </connections>
```

**Comment by Nephos** [comment]
![screenshot_20170926_181015](https://user-images.githubusercontent.com/979584/30871075-f7dd9bea-a2e5-11e7-966b-ceffef1df0a1.png)

En utilisant "signal/slots"

**Comment by Lucie-Dispot** [comment]
Fixed in e1efe998dfea20d254748b5935da38c7117ca569
